sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfig.java

```diff
@@ -115,6 +132,17 @@ public CosmosConfigBuilder enableQueryMetrics(boolean queryMetricsEnabled) {
             return this;
         }
 
+        /**
+         * Set maxDegreeOfParallelism
+         *
+         * @param maxDegreeOfParallelism value to initialize
+         * @return CosmosConfigBuilder
+         */
+        public CosmosConfigBuilder setMaxDegreeOfParallelism(int maxDegreeOfParallelism) {
```

Comment:
Noticed that we already have a builder for this, I think in this case it may be fine to change the constructors above, let's discuss this offline.

Reply:
Done.